.eslintrc.js

```diff
@@ -49,6 +50,10 @@ module.exports = {
     // allow debugger during development
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'brace-style': ["error", "stroustrup", { "allowSingleLine": true }],
+    // disable linebreak-style rule
+    'linebreak-style': 0,
+    "global-require": 0,
```

Comment:
Please remove the global-require rule, it is not within the scope of a fix for the line break styling.